ui/lib/core/addon/components/form-field.hbs

```diff
@@ -209,7 +209,18 @@
       @inputValue={{get @model this.valuePath}}
       @onChange={{this.setAndBroadcast}}
       @attrName={{@attr.name}}
+      @subText={{@attr.options.subText}}
     />
+  {{else if (eq @attr.options.editType "moreInfo")}}
```

Comment:
It feels a little weird to make this it's own attr (same thing for having an attr just to display a label or subtext like for `keyParametersLabel`)

Since this pattern is used throughout the PKI work for groups that are "hidden" behind toggles, I think it might make sense to have a separate getter in the model to get the relative form field group data based on what the `field` key is. 

For example, instead of making a new form field, or passing `hideInput` - is it possible to have a group's label, subtext and more info tag all live in one place?

One way of doing this would have a separate getter (for example `formFieldGroupData`) that associates each `fieldGroup` with its relative label, subtext and more info information, instead of storing those as model attributes. So in the template we'd do something like `{{get @model.formFieldGroupData group}}` within each form field loop and each formField group data would have a key that matches the `group` name. (Similar to have `formFieldGroups` works.

```js
keyParameters: { 
 label: 'Key parameters,
  subText: `These are the parameters for generating or validating the certificate's key material.`
  moreInfo: 'Something about more information'
}
```
Then in the hbs template we reference each group label/subText/moreInfo _outside_ the <FormField> component. 

 Let me know what you think and if it even seems possible, but I think it'd separate concerns more clearly, and clean up some of the model attrs. Happy to zoom if this explanation doesn't make complete sense

Reply:
I've tried doing something like this in the model, but the problem is no matter what the Label will show on the form field and input will show on the form field. If any form field is declared in the model the form-field.hbs file automatically creates a label and input. 

Also, I tried adding to the group (e.g. not the specific field but the group) and dang did that break everything. It has to do with field-to-attrs helper and again how form-field reads attrs. 

I'll hit you up with zoom to discuss another option.